### ISSUE TITLE
Lychee as a drop in replacement for Synology Photostation?

### ISSUE THREAD

**Issue by friesoft** [electerious/Lychee#696]
I've been searching desperately for alternatives to Synologys Photostation as it is only slowing getting improved, the codebase isn't opensource and the permission system is utter crap.

Lychee looks really awesome (ui, functionality, ...) but it is imho missing one key point to replace Photostation for me and everyone I know which owns a Synology and uses Photostation.

Our workflow with new photos is: 
- photos are automatically uploaded to the NAS (this could be worked around with other apps)
- photos are indexed automatically by Photostation as soon as they where uploaded
- photos can be modified/postprocessed/etc on the Windows share just like every other file without having to worry about the photomanagement software - as soon as I save the picture in photoshop or whatever the thumbnail is automatically updated and I can view the new image in Photostation

So the main missing thing is: I want to be able to manage photos using a normal file explorer and have the lychee library automatically updated accordingly - but it should still be possible to also work in the lychee library and have everything reflected on the filesystem.
Long story short: it shouldn't matter if I remove/edit/rename photos in the lychee ui or in the file explorer.

If all of this is already possible with lycheesync then I apologize but then it wasn't really clear to me from the provided description and it should be made more prominent as it is a cool feature. Maybe a whole wiki article or something.

Thanks for your great work :)

**Comment by friesoft** [comment]
Looks like this is related to #233 and #280... didn't find those before posting.

A tutorial/wiki article on how to reach this state would help nonetheless :)

**Comment by electerious** [comment]
This issue will be closed because I'm not actively working on Lychee anymore. The repository of Lychee will stay available, but the main development continues under the [Lychee Organisation](https://github.com/LycheeOrg). A fork maintained by the community. Important issues have been migrated. Feel free to re-add your issue(s) when necessary.
